rll/test-script.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash -ex
+# This test script is used by the regression tester in RightLink10 to verify that recipes can be
+# executed and that credentials are not echoed to the audit entry
+set -x
+echo VAR=$VAR
+echo CRED=$CRED
+printenv
+egrep 'Hi Mom' "attachments/test-attachment.txt"
```

Comment:
Should we have such a informal message?  Suggest using https://github.com/rightscale/rightlink_scripts/commit/83584e9b8353e9b4f48a16287817f8e070da8a31 which was done before this script was removed.